config.json

```diff
@@ -74,36 +74,49 @@
       "slug": "hello-world",
       "difficulty": 1,
       "topics": [
+        "IO",
```

Comment:
Oh, yeah, in the Elixir version we don't actually output that to the console. Good catch!